src/pages/staticPages/tools/oncoprinter/OncoprinterStore.ts

```diff
@@ -290,9 +291,11 @@ export default class OncoprinterStore {
                 map[next.label] = this.sampleIds.map(sampleId=>{
                     const datum = sampleToDatum[sampleId];
                     if (!datum) {
-                        return false;
+                        return CoverageStatus.UNPROFILED;
+                    } else if (datum.data.length > 0) {
+                        return CoverageStatus.ALTERED;
                     } else {
-                        return datum.data.length > 0;
+                        return CoverageStatus.UNALTERED;
                     }
```

Comment:
Hi @adamabeshouse , could I get your review on this function in the Oncoprinter? Thanks!

Reply:
looks great! except I think `CoverageStatus` should really be `AlteredStatus`. what do you think?

Reply:
That's also one thing I am gonna ask, yes I think that's great. I can change it to `AlteredStatus`

Reply:
Updated!